synfig-studio/src/gui/states/state_bline.cpp

```diff
@@ -1598,58 +1613,41 @@ StateBLine_Context::popup_vertex_menu(synfig::ValueNode_Const::Handle value_node
 		menu.remove(**i);
 
 	Gtk::MenuItem* item = nullptr;
-	Gtk::ImageMenuItem* item2 = nullptr;
 
 	BLinePoint bline_point(value_node->get_value().get(BLinePoint()));
-	#define STATE_BLINE_ADD_MENU_ITEM(title, split_angle, split_radius, icon) \
-	do {                                                                \
-		item2 = manage(new Gtk::ImageMenuItem(                       \
-				*manage(new Gtk::Image(			    \
-					Gtk::StockID(icon),		\
-					Gtk::ICON_SIZE_MENU )),			\
-				_(title)));                     \
-		item2->signal_activate().connect(                                \
-				sigc::bind(                                             \
-					sigc::mem_fun(*this,&studio::StateBLine_Context::bline_set_split_handle), \
-					value_node, split_angle, split_radius ));           \
-		item2->show();                                                   \
-		menu.append(*item2);                                             \
-	} while (false)
 
 	bool split_angle = bline_point.get_split_tangent_angle();
 	bool split_radius = bline_point.get_split_tangent_radius();
 	
 	if (split_angle && split_radius)
-		STATE_BLINE_ADD_MENU_ITEM("Merge Tangents", false, false, "gtk-connect");
+		menu.append(*create_split_merge_menu_item(_("Merge Tangents"), false, false, "gtk-connect", value_node));
```

Comment:
`gtk-connect`/`gtk-disconnect` has no replacement :(

https://docs.gtk.org/gtk3/const.STOCK_CONNECT.html

Reply:
We can create ours for now.
What about those above? They are based on current Bline Tool icon.
![merge_split_tangents_icons 0001](https://user-images.githubusercontent.com/1323987/182990071-37a81b6d-0996-4661-888b-27f313d0614b.png)
![merge_split_tangents_icons 0001](https://user-images.githubusercontent.com/1323987/182990764-bcdc0ffe-0c98-4b2d-b11b-2b9f7fcddeb1.png)
![merge_split_tangents_icons 0000](https://user-images.githubusercontent.com/1323987/182990765-a8995edd-4fd5-4907-aaa9-720cae8a408a.png)

Reply:
Last commit shows it.
I can delete it if not needed or it is bad